src/main/java/mate/academy/demo/controler/BookControler.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.demo.controler;
+
+import java.util.List;
+import mate.academy.demo.dto.BookDto;
+import mate.academy.demo.dto.CreateBookRequestDto;
+import mate.academy.demo.dto.UpdateBookRequestDto;
+import mate.academy.demo.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/api/books")
```

Comment:
add `server.servlet.context-path=/api` to application.properties

```suggestion
@RequestMapping("/books")
```